packages/x-components/src/wiring/__tests__/throttle-operator.spec.ts

```diff
@@ -0,0 +1,233 @@
+import { createResultStub } from '../../__stubs__/results-stubs.factory';
+import { throttle } from '../wires.operators';
+import { createWire } from './operators-testing-utils';
+
+describe(`testing ${throttle.name} operator`, () => {
```

Comment:
We some times do this to avoid hardcoding the name of the function in the middle of a string. This way if this function is renamed at some time (which I would probably do to `throttleWire`), the test description is also updated, while if we hardcode the `throttle` name I can guarantee that we will forget 🤣.